src/hooks/marp-options.ts

```diff
@@ -24,6 +24,7 @@ export default function useMarpOptions(
       identifier,
       marpOptions: {
         ...(opts || {}),
+        script: false,
```

Comment:
I hope this change is what you wanted. (sorry if not)

Reply:
Yes, exactly :+1: